pkg/cmd/open.go

```diff
@@ -0,0 +1,89 @@
+	"dashboard/payments":                 "https://dashboard.stripe.com/payments",
+	"dashboard/disputes":                 "https://dashboard.stripe.com/disputes",
+	"dashboard/balance":                  "https://dashboard.stripe.com/balance/overview",
+	"dashboard/payouts":                  "https://dashboard.stripe.com/payouts",
+	"dashboard/topups":                   "https://dashboard.stripe.com/topups",
+	"dashboard/transactions":             "https://dashboard.stripe.com/balance",
+	"dashboard/customers":                "https://dashboard.stripe.com/customers",
+	"dashboard/atlas":                    "https://dashboard.stripe.com/atlas",
+	"dashboard/radar":                    "https://dashboard.stripe.com/radar",
+	"dashboard/radar/reviews":            "https://dashboard.stripe.com/radar/reviews",
+	"dashboard/radar/list":               "https://dashboard.stripe.com/radar/list",
+	"dashboard/radar/rules":              "https://dashboard.stripe.com/radar/rules",
+	"dashboard/billing":                  "https://dashboard.stripe.com/billing",
+	"dashboard/invoices":                 "https://dashboard.stripe.com/invoices",
+	"dashboard/subscriptions":            "https://dashboard.stripe.com/subscriptions",
+	"dashboard/subscriptions/products":   "https://dashboard.stripe.com/subscriptions/products",
+	"dashboard/tax-rates":                "https://dashboard.stripe.com/tax-rates",
+	"dashboard/coupons":                  "https://dashboard.stripe.com/coupons",
+	"dashboard/connect":                  "https://dashboard.stripe.com/connect/overview",
+	"dashboard/connect/accounts":         "https://dashboard.stripe.com/connect/accounts/overview",
+	"dashboard/connect/transfers":        "https://dashboard.stripe.com/connect/transfers",
+	"dashboard/connect/collected-fees":   "https://dashboard.stripe.com/connect/application_fees",
+	"dashboard/orders":                   "https://dashboard.stripe.com/orders",
+	"dashboard/orders/products":          "https://dashboard.stripe.com/orders/products",
+	"dashboard/terminal":                 "https://dashboard.stripe.com/terminal",
+	"dashboard/terminal/locations":       "https://dashboard.stripe.com/terminal/locations",
+	"dashboard/terminal/hardware_orders": "https://dashboard.stripe.com/terminal/hardware_orders",
+	"dashboard/developers":               "https://dashboard.stripe.com/developers",
+	"dashboard/apikeys":                  "https://dashboard.stripe.com/apikeys",
+	"dashboard/webhooks":                 "https://dashboard.stripe.com/webhooks",
+	"dashboard/events":                   "https://dashboard.stripe.com/events",
+	"dashboard/logs":                     "https://dashboard.stripe.com/logs",
+	"dashboard/settings":                 "https://dashboard.stripe.com/settings",
+	"api":                                "https://stripe.com/docs/api",
+	"apiref":                             "https://stripe.com/docs/api",
+	"docs":                               "https://stripe.com/docs",
```

Comment:
I hate to be that one-issue-person all the time, but any chance we could alphabetize this list? The reasons I think it's a good idea are (1) if you're looking for a particular URL in the list (say you're wondering if you have to add a new one), it's easier to find when things are sorted, and (2) it gives a definitive and deterministic place to add new URLs instead of just tacking them in randomly or on the end.

Reply:
I definitely should've thought to do that, good call out.